modules/runner/src/testFixtures/java/org/apache/ignite/internal/BaseIgniteRestartTest.java

```diff
@@ -139,21 +146,93 @@ void setUp(TestInfo testInfo) {
     public void afterEachTest() throws Exception {
         var closeables = new ArrayList<AutoCloseable>();
 
-        for (IgniteServer node : IGNITE_SERVERS) {
-            if (node != null) {
-                closeables.add(node::shutdown);
+        List<String> serverNames = IGNITE_SERVERS.stream()
+                .filter(Objects::nonNull)
+                .map(IgniteServer::name)
+                .collect(toList());
+
+        List<String> partialNodeNames = this.partialNodes.stream()
+                .filter(Objects::nonNull)
+                .map(PartialNode::name)
+                .collect(toList());
+
+        log.info("Shutting the cluster down [serverNodes={}, partialNodes={}]", serverNames, partialNodeNames);
+
+        Set<String> cmgMsNodesNames = new HashSet<>();
```

Comment:
Let's be consistent in msCmg and cmgMs naming.

Reply:
done